lib/auth/windows/certificate_authority.go

```diff
@@ -42,11 +42,12 @@ type CertificateStoreClient struct {
 type CertificateStoreConfig struct {
 	// AccessPoint is the Auth API client (with caching).
 	AccessPoint authclient.WindowsDesktopAccessPoint
-	// LDAPConfig is the ldap configuration
-	LDAPConfig
-	// Log is the logging sink for the service
+	// Domain is the Active Directory domain where Teleport publishes its
+	// Certificate Revocation List (CRL).
+	Domain string
+	// Log is the loggin g sink for the service
```

Comment:
Reminder to self to fix this. Still getting used to linear switches 😅

Reply:
Fixed it :)

Reply:
Dziękuję!